pycbc/transforms.py

```diff
@@ -612,7 +612,101 @@ def inverse_jacobian(self, maps):
         ref_mass=1.4
         mchirp = maps['mchirp']
         return (2.**(-1./5) * self.ref_mass / mchirp)**(5./6)
+    
+######################################################################################################### TEST ################################################################################################################################################
+import numpy as np
+from pycbc.waveform.waveform_modes import jframe_to_l0frame
```

Comment:
Just to check (and not part of this PR). Have you checked that this code actually works? I'm not sure if it's been used for anything important ...